lib/compute/index.js

```diff
@@ -312,6 +318,96 @@ Compute.prototype.createNetwork = function(name, config, callback) {
 };
 
 /**
+ * Create a global forwarding rule.
+ *
+ * @resource [GlobalForwardingRule Resource]{@link https://cloud.google.com/compute/docs/reference/v1/globalForwardingRules#resource}
+ * @resource [GlobalForwardingRules: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/globalForwardingRules/insert}
+ *
+ * @param {string} name - Name of the rule.
+ * @param {object} config - See a
```

Comment:
I think there's context confusion! It's not "resource" as in "See the resource for explaining what this is", it's "See the definition of this resource". It's really just borrowed lingo from the official docs (see https://cloud.google.com/compute/docs/reference/latest/globalForwardingRules#resource: "A ForwardingRule resource.")

If it was "The ForwardingRule resource", that would mean I'm speaking of a particular, existing ForwardingRule.